tiledb/ml/models/_base.py

```diff
@@ -253,7 +255,7 @@ def _load_tensorboard(self, timestamp: Optional[Timestamp] = None) -> None:
                     f.write(file_bytes)
 
     @staticmethod
-    def is_v1(array: tiledb.Array) -> bool:
-        if array.schema.domain.size < np.iinfo(np.uint64).max - 1025:
-            return True
-        return False
+    def is_v1() -> bool:
+        if ModelFileProperties.TILEDB_ML_MODEL_VERSION:
```

Comment:
This is always true:
```
In [11]: ModelFileProperties.TILEDB_ML_MODEL_VERSION
Out[11]: <ModelFileProperties.TILEDB_ML_MODEL_VERSION: 'TILEDB_ML_MODEL_VERSION'>

In [12]: type(ModelFileProperties.TILEDB_ML_MODEL_VERSION)
Out[12]: <enum 'ModelFileProperties'>

In [13]: bool(ModelFileProperties.TILEDB_ML_MODEL_VERSION)
Out[13]: True
```

Fix: `return __version__ < "0.9"` (assuming the new schema will be introduced in the 0.9 release)